docs/faststore/components/molecules/product-card.mdx

```diff
@@ -719,6 +729,8 @@ Use the `ProductCard` to:
 
 `data-fs-product-card-taxes-label`
 
+`[data-fs-product-card-sponsored-label]`
```

Comment:
```suggestion
`data-fs-product-card-sponsored-label`
```

Reply:
I think we don't use `[]` 😬 
<img width="576" alt="image" src="https://github.com/user-attachments/assets/b41a9dc6-be41-417b-b59c-6ea37ef5b2d9" />

Reply:
Thanks, Fanny!